src/server/pfs/fuse/server.go

```diff
@@ -1529,17 +1534,20 @@ func (mm *MountManager) mfc(name string) (*client.ModifyFileClient, error) {
 	if mfc, ok := mm.mfcs[name]; ok {
 		return mfc, nil
 	}
-	var repoName string
+	var repoName, projectName string
 	opts, ok := mm.root.repoOpts[name]
 	if !ok {
+		// assume that the project is the default project
+		projectName = ""
 		// assume the repo name is the same as the mount name, e.g in the
 		// pachctl mount (with no -r args) case where they all get mounted based
 		// on their name
 		repoName = name
 	} else {
+		projectName = opts.File.Commit.Branch.Repo.Project.GetName()
 		repoName = opts.File.Commit.Branch.Repo.Name
```

Comment:
For consistency
```suggestion
		repoName = opts.File.Commit.Branch.Repo.GetName()
```

Reply:
Since we know `Repo` will not be nil here, I think it is unnecessary.